poetry/core/masonry/builders/builder.py

```diff
@@ -364,17 +363,12 @@ def convert_author(cls, author: str) -> Dict[str, str]:
     @classmethod
     @contextmanager
     def temporary_directory(cls, *args: Any, **kwargs: Any) -> ContextManager[str]:
```

Comment:
We should completely eliminate this `temporary_directory` method in the first commit.

Reply:
that's reared it's ugly head again after a rebase. expunged again now